superx/information_extractors/item_info_extractor.py

```diff
@@ -159,7 +159,7 @@ def standardize_weight_name(self, unit_in_hebrew):
             'גרם': ['גרם', 'גרמים'],
             'ליטר': ['ליטר', 'ליטרים', 'ליטר    '],
             'מ"ל': ['מיליליטרים', 'מ"ל', 'מיליליטר'],
-            'אין': ['יחידה', 'לא ידוע', "יח'", "'יח"]
+            'יחידה': ['יחידה', 'לא ידוע', "יח'", "'יח"]
```

Comment:
more entries have been added to this list, that do not match יחידה . see my pr. 
Is this neccessary?

Reply:
I think that showing the users "אין" as a unit of measure isn't looks well.

Reply:
ok got it, m pr was just added, so go to the script and copy the new varibles into the table

Reply:
Done.